content/blog/wazo-platform-c4-overview.md

```diff
@@ -0,0 +1,78 @@
+Title: Wazo C4 platform overview
+Date: 2019-10-28
+Authors: Aleksandar Sosic, Fabio Tranchitella
+Category: Wazo C4
+Tags: wazo
+Slug: wazo-c4-platform-overview
+Status: published
+
+# Wazo C4 platform overview
+
+## Introduction
+The Wazo platform was thought of as an IPBX programmable solution, built around the famous open source Asterisk framework. To fulfill our vision to bring a complete, open-source, cloud-ready solution for the telecommunication world we knew that the C4 (Class 4) routing and SBC were needed to scale, secure and load balance the traffic to our IPBX solution. Keeping in mind our microservice architecture we also wanted to further strip down some of the Asterisk dependant functionalities and try to use different tools.
```

Comment:
We need to switch Wazo and Wazo Platform here. Something like:

The Wazo project was thought of as an IPBX programmable solution, built around the famous open source Asterisk framework. To fulfill Wazo Platform's vision to bring a complete, open-source, cloud-ready solution for the telecommunication world we knew that the C4 (Class 4) routing and SBC were needed to scale, secure and load balance the traffic to our IPBX solution. Keeping in mind our microservice architecture we also wanted to further strip down some of the Asterisk dependant functionalities and try to use different tools.